integration-tests/ledger-signer.spec.ts

```diff
@@ -74,7 +74,7 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
         done();
       });
 
-      it('Should get the correct public key and public key hash of the Ledger for tz2 curve and default path', async (done) => {
+      it('As a User I want to verify that Ledger will provide correct public key and public key hash for tz2 curve and default path', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
Changed to
'Verify that Ledger will provide correct public key and public key hash for tz1 curve and default path'